### ISSUE TITLE
`returnDeleted` is ignored by Swagger generator

### ISSUE THREAD

**Issue by prmtl** [nestjsx/crud#268]
When supplying options for crud:
```typescript
CrudConfigService.load({
  routes: {
    deleteOneBase: {
      returnDeleted: false
    }
  }
});
```
Swagger still shows that DELETE endpoint returns some data which is confusing.

**Comment by omerxx** [comment]
Happened to me too, but I think it's ignored completely not just by swagger. The global options simply have no effect

**Comment by zMotivat0r** [comment]
fixed in `4.3.0-beta`
